data/table.txt

```diff
@@ -8815,12 +8895,14 @@
 喥 喥 1 1 0 0 1 0 0 0 0 rite rite NA 10583
 喦 喦 1 1 0 0 1 0 0 0 0 rrru rrru NA 10568
 喧 喧 1 1 0 0 1 0 0 0 0 rjmm rjmm NA 19237
-喨 喨 1 1 0 0 1 0 0 0 0 ryru ryrn,ryru,xryrn NA 10584
+喨 喨 1 1 0 0 1 0 0 0 0 ryru ryrn,xryrn NA 10584
+喨 喨 1 1 0 0 1 0 0 0 0 NA ryru NA 0
```

Comment:
@yookoala [said](https://github.com/Cangjians/libcangjie/commit/d9d2e686ce6776ec67e89b4256dce42359597d84#r29099729):

> "ryrn" do not give the character "喨" on MacOS. "ryru" is the standard mapping.

---

Should we remove `ryrn` and `xryrn` before running the script, then?

Reply:
So Wan Leung just said that Mac OS X uses the short-lived Cangjie version 4, not version 5. :open_mouth:

Discussing this with @yookoala, we decided to ignore all those comments about Mac.